pkgs/by-name/su/sudo-rs/package.nix

```diff
@@ -80,11 +82,7 @@ rustPlatform.buildRustPackage (finalAttrs: {
   '';
 
   passthru = {
-    updateScript = nix-update-script {
-      extraArgs = [
-        "--version-regex=^v([0-9]+\\.[0-9]+\\.[0-9])$"
-      ];
```

Comment:
I see no problematic releases in their history, so removed this instead of fixing.

Reply:
They used to make separate releases including generated man pages that had a `-man` suffix, and those were continuously causing the update bot to create PRs. But I don't see those releases anymore, so maybe they don't do that anymore.